include/xten/Dialect/XTenNN/IR/XTenNNOps.td

```diff
@@ -219,16 +221,30 @@ def XtenNN_MishOp: XTenNN_Op<"mish", [Pure, TosaExtension, ElementwiseUnary, Sam
   let assemblyFormat = [{ operands attr-dict `:` functional-type(operands, results) }];
 }
 
-def XtenNN_EluOp: XTenNN_Op<"elu", [Pure, TosaExtension, ElementwiseUnary, SameOperandsAndResultElementType]> {
-  let summary = "Calculate the elu operation of the given input tensor element-wise";
+def XtenNN_RoundOp: XTenNN_Op<"round", [Pure, TosaExtension, ElementwiseUnary, SameOperandsAndResultElementType]> {
+  let summary = "Calculate the round operation of the given input tensor element-wise.";
   let description = [{
-    Calculate the elu operation (Exponential Linear Unit) of the given input tensor element-wise.
-    ELU(x) =  x,                  if x >  0
-              alpha * (exp(x)-1), if x <= 0
+    Calculate the round operation of the given input tensor element-wise.
+    In case of halves, the rule is to round them to the nearest even integer as does the 'round' operation in torch and onnx.
```

Comment:
ONNX specifies the edge cases for NaN, Infitity and +0/-0 but torch does not afaics. Which do we do here?

Reply:
I think we can assume it is a one-to-one translation from Torch to ONNX (didn't test it but it makes sense). In that case, Torch would be similar to ONNX, and I am thinking that LLVM::RoundEven is dependent on how the target implements.